osp/views/form.py

```diff
@@ -22,11 +22,11 @@ def get_permissions(self):
 
     def get_queryset(self):
         queryset = self.queryset
-        status = self.request.query_params.get('status', None)
+        status = self.request.GET.getlist('status', None)
```

Comment:
Use this:
https://medium.com/@AADota/spring-passing-list-and-array-of-values-as-url-parameters-1ed9bbdf0cb2

We should never pass data inside body of GET request.